### PR TITLE
Added travis.yml file for automating building and running django server

### PR DESCRIPTION
**Pull request by clundin55**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Django==2.1.1
```
Comment (aniehuser) [inline requirements.txt:1]: need to add `djangorestframework` to pip install

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "3.5"
+  - "3.6"
+  - "3.7" 
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
```
Comment (aniehuser) [inline .travis.yml]: should this be `pip3`?

File: requirements.txt (hunk 1 above)
Comment (clundin55) [reply requirements.txt:1]: added

File: .travis.yml (hunk 2 above)
Comment (clundin55) [reply .travis.yml]: possibly. I think this next commit is going to build and then ill check if the tests ran

**Review by aniehuser — APPROVED** [review]
LGTM :)

**Comment by aniehuser** [comment]
CVE-2018-16984 More information
high severity
Vulnerable versions: >= 2.1, < 2.1.2
Patched version: 2.1.2
An issue was discovered in Django 2.1 before 2.1.2, in which unprivileged users can read the password hashes of arbitrary accounts. The read-only password widget used by the Django Admin to display an obfuscated password hash was bypassed if a user has only the "view" permission (new in Django 2.1), resulting in display of the entire password hash to those users. This may result in a vulnerability for sites with legacy user accounts using insecure hashes.

@clundin55  django 2.1 is highly insecure, we should only be testing and using greater than 2.1.2

**Pull request merged**